### PR TITLE
Adding getClientHeaders

### PR DESCRIPTION
**Pull request by haugli92**
I have added so that it is possible to access headers from connections.

### CODE REVIEW

File: src/Server.php [hunk 1]
```diff
@@ -124,6 +124,7 @@ public function __construct(
         $this->host = $host;
         $this->port = $port;
         $this->ipcSocketPath = $ipcSocketPath;
+        $this->timers = new TimerCollection();
```
Comment (nekudo) [inline src/Server.php:127]: Why is this line moved? Seems to have nothing to do with the headers...

**Review by nekudo — CHANGES_REQUESTED** [review]

File: src/Server.php (hunk 1 above)
Comment (haugli92) [reply src/Server.php:127]: I added this fix to my fork, to make timers work. Ref #40.
So this change will no make any difference.
